subworkflows/nf-core/mapping.nf

```diff
@@ -77,14 +79,31 @@ workflow MAPPING {
     bam_bwa.map{ meta, bam ->
         meta.remove('read_group')
         meta.id = meta.sample
+        // groupKey is to makes sure that the correct group can advance as soon as it is complete
+        // and not stall the workflow until all pieces are mapped
         def groupKey = groupKey(meta, meta.numLanes * params.split_fastq)
         tuple(groupKey, bam)
         [meta, bam]
-    }.groupTuple() //groupKey above is somehow makes sure, the workflow doesn't stall until all pieces are mapped, but that the correct group can advance as soon as it is complete
+    }.groupTuple()
     .set{bam_mapped}
 
-    // STEP 1.5: MERGING AND INDEXING BAM FROM MULTIPLE LANES // MarkDuplicates can take care of this
+    // MarkDuplicates can handle multiple BAMS as input, so no merging/indexing at this step
+    // Except if and only if skipping MarkDuplicates
+
+    if (skip_markduplicates) {
```

Comment:
Ah yes, adding it here makes sense. I struggled finding a logical way on how to sort all the different ways how to walk through to base recalibration